### ISSUE TITLE
php7.4(alpha) deprecated messages

### ISSUE THREAD

**Issue by bradymiller** [rospdf/pdf-php#131]
Noted the following deprecated messages when using this library in php7.4(alpha):
```
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2386
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2387
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2388
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2389
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2390
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2391
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2407
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2408
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2409
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2410
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2411
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 2412
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 3091
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 3097
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 3099
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 3100
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 3102
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 3103
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/Cpdf.php on line 3104
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTFsubset.php on line 494
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 927
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 930
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1106
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1118
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1119
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1130
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1131
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1146
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1147
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1148
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1149
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1182
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1197
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1198
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1199
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1200
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1225
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1226
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1257
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1257
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1292
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1306
NOTICE: PHP message: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /usr/share/nginx/html/openemr/vendor/rospdf/pdf-php/src/include/TTF.php on line 1307
```
I'm happy to do a blind fix on this and submit a PR (just let me know), but can't really test it.

**Comment by ole1986** [comment]
Feel free to do so. I am going to merge it once my php 7.4 setup is ready to test.

**Comment by ole1986** [comment]
Fixed with PR #136
